src/routes/auth/create-new-user.e2e.test.js

```diff
@@ -13,7 +13,8 @@ describe('POST /auth/create-user', () => {
 				firstName: faker.name.firstName(),
 				lastName: faker.name.lastName(),
 				email: faker.internet.email(),
-				password: faker.internet.password()
+				password: faker.internet.password(),
+				profilePictureUrl: faker.internet.url()
```

Comment:
we don't need to add this to each error case, just the error case that you're testing and the success one is cool

Reply:
okay

Reply:
done